code/modules/library/random_books.dm

```diff
@@ -10,82 +10,80 @@
 
 /obj/item/book/random
 	icon_state = "random_book"
-	var/amount = 1
-	var/category = null
+	var/random_category = null
+	var/random_loaded = FALSE

```

Comment:
[Please add DMDOC which matches the required standard](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md)